package.json

```diff
@@ -34,11 +34,11 @@
   "devDependencies": {
     "husky": "^4.3.6",
     "webpack": "^4.42.0",
-    "webpack-cli": "^4.3.1"
+    "webpack-cli": "^4.7.0"
   },
   "dependencies": {
-    "@eightshift/frontend-libs": "https://github.com/infinum/eightshift-frontend-libs#develop",
-    "@wordpress/server-side-render": "^1.16.0"
+    "@eightshift/frontend-libs": "git+https://github.com/infinum/eightshift-frontend-libs.git#develop",
+    "@wordpress/server-side-render": "^1.21.3"
```

Comment:
this should work without this lib because it is instaled in the core. can you please test

Reply:
Will test in another PR, just trying to get release zipping working now on releases.